CodeStyleExercise/ColorsViewController.swift

```diff
@@ -8,55 +8,52 @@
+    UITextFieldDelegate
 {
-
-    class constants {
-        static let DFLTNUMCOLORS:Int=12
-    }
-
+    
     @IBOutlet weak var collectionView:UICollectionView!
     @IBOutlet weak var colorCountTextField: UITextField!
 
-    var ct: Int? = 12
+    var colorCount = 12
 
     override func viewDidLoad() {
         super.viewDidLoad()
-        self.collectionView.register(UINib(nibName: "ColorsCollectionViewCell", bundle: nil), forCellWithReuseIdentifier: "Cell")
-
+        self.collectionView.register(UINib(nibName: "ColorsCollectionViewCell",
+                                            bundle: nil),
+                        forCellWithReuseIdentifier: "Cell")
         self.colorCountTextField.placeholder = "12"
     }
-
-    override func didReceiveMemoryWarning() {
-        super.didReceiveMemoryWarning()
+    
+    private func randomColor() -> UIColor {
+        let red = CGFloat(arc4random_uniform(256))/255
+        let green = CGFloat(arc4random_uniform(256))/255
+        let blue = CGFloat(arc4random_uniform(256))/255
+        
+        return UIColor(red: red, green: green, blue: blue, alpha: 1.0)
     }
 
-    // MARK: - Collection View Data Source Methods
+    // MARK: UICollectionViewDataSource
 
-    func collectionView(_ collectionView:UICollectionView, numberOfItemsInSection section : Int) -> Int{
-        return ct!
+    func collectionView(_ collectionView:UICollectionView, numberOfItemsInSection section : Int) -> Int {
+        return colorCount
     }
 
     func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
-        var c = collectionView.dequeueReusableCell(withReuseIdentifier: "Cell", for: indexPath) as! ColorsCollectionViewCell
-//        let color = UIColor.random
-//        c.backgroundColor = color
-//        c.hexColorLbl.text = color.description
-        c.configureCellWithColor(color: random_ui_color())
-        return c
+        let cell = collectionView.dequeueReusableCell(withReuseIdentifier: "Cell", for: indexPath) as! ColorsCollectionViewCell
+        cell.configureCellWithColor(color: randomColor())
+        
+        return cell
     }
 
-
-
-
-    //////////////////////////////////
-    // MARK: - UITextFieldDelegate ///
-    //////////////////////////////////
+    // MARK: UITextFieldDelegate
 
     func textField(_ textField: UITextField, shouldChangeCharactersIn range: NSRange, replacementString string: String) -> Bool {
-        if string.characters.count>0{
+        if string.characters.count > 0 {
             if (textField.text! as NSString).replacingCharacters(in: range, with: string).characters.count <= 2 {
-                var str = string.components(separatedBy: NSCharacterSet.decimalDigits.inverted).joined() // Filters string by removing non-decimal digits
+                let str = string.components(separatedBy: NSCharacterSet.decimalDigits.inverted).joined() // Filters string by removing non-decimal digits
```

Comment:
Consider picking a more descriptive name for the string so we don't need the comment.